pyam/core.py

```diff
@@ -879,11 +880,16 @@ def aggregate(self, variable, components=None, method='sum', append=False):
         method : func or str, default 'sum'
             method to use for aggregation,
             e.g. :func:`numpy.mean`, :func:`numpy.sum`, 'min', 'max'
+        recursive : bool, default False
+            iterate recursively over all sublevels of `variable`
         append : bool, default False
             append the aggregate timeseries to `self` and return None,
             else return aggregate timeseries as new :class:`IamDataFrame`
         """
```

Comment:
I realized that there the implementation assumes that all nan are 0 - I guess that this is ok (for now), but it should be added explicitly in the docs.
```suggestion

        Notes
        -----
        The aggregation function interprets :any:`numpy.nan` as 0.
        """
```